atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -379,17 +403,48 @@ public void hostsWithDifferingTopologiesTrackedAsAbsentHost() {
         assertThat(absentHostTracker.returnPool(CASS_SERVER_2)).isPresent();
     }
 
+    @Test
+    public void doesNotThrowWhenPoolDoesNotChange() {
+        when(config.autoRefreshNodes()).thenReturn(true);
+        setCassandraServersTo(CASS_SERVER_1, CASS_SERVER_2, CASS_SERVER_3);
+        CassandraClientPoolImpl pool = createClientPool();
+        assertThatNoException()
+                .isThrownBy(() -> pool.setServersInPoolTo(ImmutableMap.of(
+                        CASS_SERVER_1,
+                        CassandraServerOrigin.CONFIG,
+                        CASS_SERVER_2,
+                        CassandraServerOrigin.CONFIG,
+                        CASS_SERVER_3,
+                        CassandraServerOrigin.CONFIG)));
```

Comment:
this one is probably just an Atlas style thing / for consistency: we normally use the I think equivalent `assertThatCode(() -> ...).doesNotThrowAnyException()`. I do actually quite like this way though we should probably be consistent.

Reply:
woops, yup sorry about that!